test/ColdResumptionTest.cpp

```diff
@@ -93,6 +95,33 @@ class HelloResumeHandler : public ColdResumeHandler {
 };
 }
 
+std::unique_ptr<rsocket::RSocketClient> createResumedClient(
+    folly::EventBase* evb,
+    uint32_t port,
+    ResumeIdentificationToken token,
+    std::shared_ptr<ResumeManager> resumeManager,
+    std::shared_ptr<ColdResumeHandler> coldResumeHandler) {
+  auto retries = 10;
+  while (true) {
```

Comment:
What's the reason of retry? Is it because of having multiple threads?
Can we utilize the eventbases and manually loop them to have the treads be in sync? 
Afaik there is a class,  `folly::DeterministicSchedule`, that we can utilize.

Reply:
createResumedClient() could fail if the old transport hasn't disconnected on the server side.  It is intentional to have some failure cases so that we can test the error paths.